webapp/src/createFunction.tsx

```diff
@@ -151,9 +151,9 @@ export class CreateFunctionDialog extends data.Component<ISettingsProps, CreateF
             }
         ];
         const types = this.getArgumentTypes().slice();
-
+        const classes = ["createfunction", this.props.parent.rootClasses.filter((e: string) => e != "full-abs").join(" ")].join(" ");
```

Comment:
```suggestion
        const classes = sui.cx(["createfunction", ...this.props.parent.rootClasses.filter((e: string) => e != "full-abs")]);
```

we have a helper for this https://github.com/microsoft/pxt/blob/70c8a663c90fc5a14fc5cb3a05343fee785644c7/webapp/src/sui.tsx#L32-L34

(along the lines of the comment above, though, I'd suggest making this a getter instead of just a property; `this.props.parent.rootClasses()` or something like that, keep the logic for getting all the classes in one place / minimize chance of someone mutating it by accident and breaking things)